contrib/hack/kubectl.sh

```diff
@@ -17,6 +17,12 @@ set -e
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
 
+if [[ "${WITH_TTY:-}" -eq 1 ]]; then
```

Comment:
If we're going to go this path then the default needs to be to have `-ti` enabled and let's make jenkins turn it off.
I want people to be able to run kubectl to test stuff out (manually) w/o having to set additional flag.s  The point of this script is to be just like the normal/stand-alone kubectl as much as possible.